app/lib/publisher/models.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:gcloud/db.dart' as db;
+
+/// Canonical publisher data.
+@db.Kind(name: 'Publisher', idType: db.IdType.String)
+class Publisher extends db.ExpandoModel {
+  /// The associated domain name of the publisher.
+  String get domainName => id as String;
```

Comment:
Perhaps it's better to call this `publisherId` everywhere, and then leave in a comment that this is the verified domain?

Who knows, if people don't verify domains and don't want to use this, it could happen that in some future we simply allow creation of arbitrary publisherIds... We can't tell if this will be used or not, maybe users find domain verification too hard.

Reply:
Maybe we should start with an UUID identifier, and store the verified domain as a separate field?